docs/faq/events.md

```diff
@@ -67,15 +67,15 @@ If you are adding a listener to your element, from inside your element, you shou
 ```js
 constructor() {
     super();
-    this.addEventListener('done', this.handleDone.bind(this));
+    this.addEventListener('done', this.handleDone);
 }
 handleDone(event) {
     this.done = true;
     // Do the work you'd like to complete when "done" here.
 }
 ```
 
-Side note: as you will be adding this listener manually, you _will_ need to ensure that `this` is bound to your liking. See the `.bind()` syntax above.
+Side note: as long as your event handler is a method of your custom element, there is no need to manually `bind` it. When calling `addEventListener`, the value of `this` inside the event handler [is a reference to the element](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#The_value_of_this_within_the_handler).
```

Comment:
With this correction, I'm not sure we should side note this...even for me, having written this, it brought confusion in reviewing your update 🤦  In that it doesn't break if you were to bind, I think we should leave the idea of bind out of this sectional altogether. Thoughts?

Reply:
I can buy that! Although I don't know if it is well known that `this.addEventListener` auto-binds to whatever the event is attached to. Do you think it is better to call it out, given that the section below does call out that bind is required when attaching the event to a different element?